sarahricker/thinkery
# initial footer development

## JIRA Ticket

[A link to the relevant JIRA ticket](https://wpengine.atlassian.net/browse/THNK-47?atlOrigin=eyJpIjoiNTRjOGRlNzY2NzEyNDI4NWI1OTVkZDU0MGNhYTI5MzEiLCJwIjoiaiJ9)

## Description

This is a new feature branch for the Footer designed and built with Widget Navs. This is an initial design and open to modifications of optimizations.

## Impacted Areas in Application

List general components of the site that this PR will affect:

* footer-widgets
* site-footer

## Deploy Notes

Notes regarding deployment the contained body of work.
If you need to change something in WordPress after deploying, make the necessary steps crystal clear.

## Steps to Test or Reproduce

1. Pull down the changes
2. Navigate into the `/wp-admin` and login
3. Once in the backend you will want to go to the **Appearance -> Widgets**
4. In **Footer 1** add a **Text Widget** and toggle the editor window tab from _Visual_ to _Text_ and in the input box below copy and past this code:

```
<img class="alignnone size-medium wp-image-8" src="http://thinkery.wpengine.com.test/wp-content/uploads/2019/05/logo-300x79.png" alt="" width="300" height="79" />
1830 Simond Ave
Austin, Texas 78723

512-469-6200
info@thinkeryaustin.org
<div id="fa-icons"><i class="fab fa-facebook-square"></i><i class="fab fa-twitter"></i><i class="fab fa-linkedin-in"></i><i class="fab fa-youtube"></i><i class="fas fa-rss"></i></div>
```
The end result for **Footer 1** should look like this: https://imgur.com/a/qc1fe7I

5. In **Footer 2** add a **Nav Widget** and input the Title in all caps **EXPLORE** and then open the down for the Select Menu option and choose the Footer Nav option. The end result should look like this: https://imgur.com/a/P8T8faw

6. Replicate **Step 5** in Footer 3 changing the Title to **VISIT** instead of **EXPLORE**
7. Replicate **Step 5** in Footer 4 changing the Title to **SUPPORT** instead of **EXPLORE**
8. Replicate **Step 5** in Footer 5 changing the Title to **ABOUT** instead of **EXPLORE**
9. Save Changes
10. To add the site logo into the site header from the **wp-admin** backend go into the **Appearance -> Customizer -> Site Identity** and click the **Select-Logo** box which will take you to the Media Library. From the Media Library search for **logo.png** and choose this as the site logo, skip cropping on the next page it brings you to.
11. Back in the **Customizer -> Site Identity** page set the **Logo Width** to **168** 
12. Publish changes.

## Random GIF

Include one random GIF to "pay" your reviewers in humor.

![Thanks for reviewing my code!](https://media.giphy.com/media/Y4OxQ9b9YxyuQi8Hgg/giphy.gif)
